ada_moveit/calib/default/calib1/camera_calib.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
```

Comment:
See comment on `calib2`:

Why is this an XML launch file instead of a Python one that can be called from demo? 

Best to move it over, since it should be quick to do so.